### ISSUE TITLE
Share learning part of the code?

### ISSUE THREAD

**Issue by MKLOL** [rloniello/RT-Salt-Bettor#1]
Hey can you share the learning part of the code?

I literally wrote yesterday a python script that does the same thing as your app (show probabilities), and I wanna modify your learning to run on my data (i have less data, but I also include the map, tho i messed up a bit during collection lol, so I'm curious on differences).

**Comment by rloniello** [comment]
Yes, the data and the ML model generated with "Create ML" are in the resources folder as "RTClassifier". The match data collected is also in this folder both as a Numbers file and a CSV. The classifier is specific to CoreML / iOS it is a complied version of a machine learning model that was exported after learning. 

The model is a binary classification using a Random Forest. You can find a python example here: https://medium.com/@hjhuney/implementing-a-random-forest-classification-model-in-python-583891c99652
There are many other examples available online. 

You may also find it helpful to watch WWDC videos on CoreML and CreateML. If you need additional help let me know here.

**Comment by MKLOL** [comment]
I already have a python ML just wanted to train to compare. 

Btw I think ur ML model is a bit overfitted. 

Arty mmr: 2150
vP 2150 MMR gives 58% loss. vP 2120 mmr gives 62 loss. 

Lots of contradictions like this. Going from 2006 to 2007 jumps from 50/50 to 55/45 (arty loss). Then if you go up to 2019  arty is now favored 54%. 

Pretty obvious that any good ML model shouldn't take these kinds of jumps since bigger MMR means better player on average. My linear regression model doesn't have these big jumps in problbailty. My bet is forests are either overfitted (too many nodes), or it might just be a bad model for these types of MMR based predictions.
